packages/fluentui/docs/src/examples/components/Card/Usage/CardExamplePreview.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button, Flex, Image, Text, Card, Checkbox } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const CardExamplePreview = () => (
+  <Card compact>
+    <Card.TopControls>
+      <Checkbox aria-label="test checkbox" />
```

Comment:
We have `aria-label` here, but we don't have `alt` on `Image`s 🤔

Reply:
I've fixed only errors I saw :)
Will update this as well

Reply:
Oh, I guess it's because by default we add `aria-hidde: true` if the image doesn't have a label

Reply:
So, will leave it like that :)